### ISSUE TITLE
Build report for identifying Knack <> 311 variances

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#16539]
Build a report to identify discrepancies between Knack and 311 for Arterial Management and Signs & Markings (users processing 311 SRs in Knack).

The report should: 
- Identify 311 CSRs that are missing from Knack service requests
- Identify 311 CSRs that are closed in Knack and are open in 311

Source Socrata Datasets:
- [Signs & Markings Service Requests](https://datahub.austintexas.gov/Transportation-and-Mobility/Signs-and-Markings-Knack-311-Service-Requests/ji6n-8jff/about_data)
- [Arterial Management Mobility Management Center Issues](https://data.austintexas.gov/Transportation-and-Mobility/Mobility-Management-Center-Issues/v7vh-gbi6/about_data)
- [Transportation & Public Works 311 Service Requests](https://datahub.austintexas.gov/Transportation-and-Mobility/Transportation-and-Public-Works-311-Service-Reques/38mr-dwji/about_data)

**Comment by ChristinaTremel** [comment]
@Charlie-Henry the [Signs & Markings Service Requests](https://datahub.austintexas.gov/Transportation-and-Mobility/Signs-and-Markings-Knack-311-Service-Requests/ji6n-8jff/about_data) dataset has been created and this report is ready to be built! I'll bring it up at the dev + apps sync today.

**Comment by Charlie-Henry** [comment]
@ChristinaTremel here's my rough draft of the power BI dashboard. I'm seeing a few missing CSRs within the last 24 hours but I think that is mainly caused by the time that the 3-1-1 report was run versus the time the ETL was run from Knack. Most of the mismatched statuses are also caused by this delay but could still be useful for debugging things?

https://app.powerbigov.us/links/GljHX9ZTiw?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare

**Comment by ChristinaTremel** [comment]
Woah, this is awesome! I'll research some of the SRs that are showing up on the missing and mismatched statuses to spot check the info! I think the time difference between when each report is run definitely makes sense for the newer SRs on the report.

**Comment by Charlie-Henry** [comment]
Let me know if you have any changes you'd like to make, it's kinda barebones but maybe that's ideal @ChristinaTremel

**Comment by ChristinaTremel** [comment]
I was able to review the dashboard and REALLY like it. I think it should be kept super simple and barebones because we can then just leverage this data and communicate with 311 when we have things missing or we have conflicting SR statuses between the two systems. 

If there's time today, I might try to discuss this during the data science sprint planning because I'd love to get y'alls perspective if maybe we should filter out any SRs submitted the current date. I would definitely expect to see discrepancies the day SRs are submitted because of the delays between Knack <> 311 and because of the time difference like you said before!

**Comment by ChristinaTremel** [comment]
@Charlie-Henry here are the changes we talked about in Data Science! 
#### Dashboard:
- Add cards of count of current missing count and current mismatched count
#### Missing SRs
- Add filter check boxes that default filter out the CLOSED status missing SRs, with the option to unfilter if needed
#### Mismatched Statuses:
- Instead of using the created date, bring in and sort by the modified date on the AMD and SMD datasets

**Comment by Charlie-Henry** [comment]
@ChristinaTremel, I've addressed these changes and moved it to the production workspace:

https://app.powerbigov.us/links/8XyaXQHNOk?ctid=5c5e19f6-a6ab-4b45-b1d0-be4608a9a67f&pbi_source=linkShare

**Comment by Charlie-Henry** [comment]
I think we're good to close this one out? @ChristinaTremel

**Comment by ChristinaTremel** [comment]
So close! I was able to check and compare more of the records in 311 <> Knack and the last thing I want to adjust is:
- For the mismatched status SRs: can we filter out records modified both today and yesterday from the report? 

I compared the statuses and anything that shows errors/mismatched statuses from 5/22 are correct now. So I think the variance is just because of the timing that the reports were run like you had said before! 

After that we are GOOD to close this out! Thank youuuu ✨
